features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -355,6 +382,21 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     };
 
+    const hybridFlowConfigValuesChangeListener = (tempForm: Map<string, FormValue>): void => {
+
+        const withPredicate = (val: string): boolean => val === HYBRID_FLOW_ENABLE_CONFIG ;
+
+        if (tempForm.has(HYBRID_FLOW_ENABLE_CONFIG)) {
+            const value: string[] = tempForm.get(HYBRID_FLOW_ENABLE_CONFIG) as string[];
```

Comment:
Since the type of `value` is a string array, let's change the variable name to `values`

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6271/commits/11295ba4d8273091b1934290f09ab0ab4d02ac0d